### ISSUE TITLE
"block-closing-brace-newline-after" throws error when "stylelint-disable-line" is used

### ISSUE THREAD

**Issue by Nirazul** [stylelint/stylelint#1008]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I wanted to disable a statement for one single line. As a result, another rule started throwing errors, which defies the purpose of a granular deactivation of stylelint rules.

> Which rule, if any, is this issue related to?

`block-closing-brace-newline-after` and probably also `block-closing-brace-space-after`

> What CSS is needed to reproduce this issue?

e.g.

```css
.a {} /* stylelint-disable-line block-no-empty */
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "block-closing-brace-newline-after": "always",
  }
}
```

> Which version of stylelint are you using?
e.g. `4.3.2`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

gulp-stylelint

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
test.css
  141:15  error  Expected newline after "}"   block-closing-brace-newline-after
```

**Comment by ntwb** [comment]
> Which rule, if any, is this issue related to?
> `block-closing-brace-newline-after` and probably also `block-closing-brace-space-after`

Confirmed `block-closing-brace-newline-after`

Could not reproduce the issue with `block-closing-brace-space-after`, I'd say this works as expected, as `.a{}/* stylelint-disable-line block-no-empty */` fails, because there is no space after the closing brace `}` and `.a{} /* stylelint-disable-line block-no-empty */` passes as there is a space after the closing brace `} `.

I've added tests in #1009 for both the above rules

**Comment by jeddy3** [comment]
Done in #1009
